src/components/media/GalleryImageCard.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react'
+import Image from 'next/image'
+import clx from 'classnames'
+import Card from '../ui/Card/Card'
+import TagList from '../media/TagList'
+import { MobileLoader } from '../../js/sirv/util'
+import { MediaWithTags } from '../../js/types'
+import { getUploadDateSummary } from '../../js/utils'
+import { PostHeader } from '../home/Post'
+
+const MOBILE_IMAGE_MAX_WIDITH = 600
+
+interface GalleryImageCardProps {
+  header?: JSX.Element
+  mediaWithTags: MediaWithTags
+  onImageClick?: (event: React.MouseEvent<HTMLImageElement>, mediaWithTags: MediaWithTags) => void
+}
+
+/**
+ * Image card for the gallery page
+ */
+export const GalleryImageCard = ({
```

Comment:
Visually these cards are very similar to RecentMedia cards.  Let's merge after you rebase and refactor later.